### ISSUE TITLE
Crosscope - Founding Team Member/Senior Data Scientist - Remote USA

### ISSUE THREAD

**Issue by jayendrashinde91** [TechnologyMasters/jobs#334]
![image](https://user-images.githubusercontent.com/17430907/94655388-1f0f5c00-031c-11eb-9c12-bda9aa07bc50.png)
# About Crosscope Inc.

Founded by a team of Stanford University and GE Healthcare researchers, Crosscope is a medical AI software company developing cutting-edge deep learning technology for pathologists to speed up diagnosis and deliver data-driven insights through imaging biomarkers in rendering a more accurate and reliable cancer diagnosis.

Crosscope's mission is to radically transform patient care by unlocking the power of AI.

Our company is built on principles to cultivate curiosity, diversity and inclusiveness guided by our core values of:

- Passion: A passion for cancer care fuels our innovation
- Collaboration: Multi-disciplinary team players motivated by the same goal
- Transparency: Honesty and openness builds trust
- Continuous Learning: Constant personal and professional growth 

# What You'll get

### Salary Expectation

- $75k – $90k • 1.8% – 2.2%
- Contract / Full Time

### Benefits

- Founding Team member role
- Flexible work environment
- Collaborative and creative environment
- Be part of the journey of a fast growing Medical AI SaaS Company

### Location

- Mountain View, California
- Work from home till the COVID19 pandemic settles

## What You'll Do _(Job Description)_

Work on something big and make a huge impact on patient lives....

- Founding team member duties:
a. Ensure that the Product roadmap is in alignment with Crosscope's vision
b. Create & keep a close eye on your KPIs
c. Build and Engage your Team
d. Participate in executive meetings to define product roadmaps

- Data Scientist role:
a. Leading the development of advanced image analysis techniques (segmentation, classification, object detection) and best in-class AI/DL algorithms that directly address important biological and clinical questions
b. Close interactions with data engineers, subject matter experts (SMEs), and business stakeholders to select, optimize and apply algorithms to advance our digital pathology-based biomarker development activities
c. Update and report relevant results to interdisciplinary project teams and stakeholders
d. Maintain a keen awareness of developments in data and cloud infrastructures and state-of-the-art of AI/ML/DL algorithms and research results
e. Evaluation and coordination of both academic and startup collaborations
f. Represent the digital pathology team through interactions with our key investigators and public presentations
g. Participate and thrive in a fast-paced, interactive and team-oriented culture

### Must Have

• MS degree in computer science, biomedical imaging or in a field related to signal/image processing, pattern recognition, and/or computer vision  or equivalent practical experience
•  Experience developing, training and evaluating classical machine learning models such as SVMs, Random Forests, Gradient Boosting, CNN, FCN, ResNet, GAN, etc.
•  Experience developing, training and evaluating deep-learning models using public deep learning frameworks (e.g. PyTorch, Keras, and Tensorflow)
• 5+ years of work experience in Machine Learning or Artificial Intelligence
• Desire to advance the medical field with Machine Learning
• Fluent with data processing and management of large-scale imaging and medical data/formats
• Prior modeling experience in computer vision
• Extensive experience with Python and other general purpose programming languages
• People skills - you must be a team player with excellent communication skills and a collaborative mindset

### Nice to Have

• PhD degree in Computer Science, Artificial Intelligence, Machine Learning, or related technical field
• Record of meaningful scientific publications in high impact, technical or medical journals/conferences
• Experience in cloud architecture, serverless computation and AWS Cloud Platform. Familiarity with AWS Core services: EC2, SQS, RDS, ELB, IAM

### Relevant Experiences

- Healthcare industry knowledge / experience

## How to apply

Please send an email to [info@crosscope.com](mailto:info@crosscope.com) with your resume, use 'Senior Data Scientist' as subject

---

## Meta

- [x] Full Time
- [x] Contract
- [x] Internship
- [x] Remote Regional
- [x] Remote OK
